### PR TITLE
Enhancements and Fixes for Windows Course

### LINKED ISSUE #493: GitHub for Windows Users - Future Additions
## Overview
With the [GitHub for Windows Users](https://github.com/github/training-kit/pull/488) about to go live, I am creating this issue for students and other interested parties to post relevant ideas to enhance the class before working on the next iteration of the course. 

#### Reasoning
From a previous conversation with @saraford on Slack, the following ideas were suggested: 
- Example of branching, show on whiteboard, GH desktop, then within VS
- Why git
- Setup walkthrough
- Git shell demos (show that it exists, and you don't have to login)
- Merge conflicts
- Demo beyondcompare / merge conflict resolution tools

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: N/A

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

### LINKED ISSUE #573: Reviewing GitHub for Windows | Add Content
## Overview

Review the content of the GitHub for Windows course and potentially add content based on #493. 

@hollenberry since reviewing / adding content can potentially be duplicative, I'm going to open this issue and identify some of the things I find and I will open the pull request to implement both the edits and new content.

- [x] Fix this: the links to other courses on the first page don't work.
- [x] Fix this: `want to attributed to the commits you make.` (Configure Git)
- [x] Fix this: `define your Carriage Return Line Feed (crlf)` add "settings" at the end (Configure Git)
- [x] Fix this: `Thankfully, Git is comes to the party equipped` (Configure Line Endings)
- [x] Discussion: This is a Windows based course, should we even include this: `These instructions assume you are using Windows, if you are using macOS or a flavor of Linux, use the following: git config --global core.autocrlf input.` (Configure Line Endings)
- [x] Fix this: `a change in OS X` change to `macOS` (Configure Line Endings: TMW)
- [x] Fix this: ` enter it agin` (Store Your Password)
- [x] Fix this: `If you haven’t defined a core.editor for git to use,` capitalize Git (vi Basics)
- [x] Remove this: `, or :q to quit without saving your commit message.` Step 2 on Saving Commit (vi Basics)
    - [x] Add it as a note under Saving the Commit
- [x] Fix this: ` VI to craft yo` (Configure Other Editors)
- [x] Fix this: `In some instances, you will need to use the installation path of Atom when setting the config settings.` Make this a note (Configure Other Editors)
- [x] Fix this: `Tools > Extensions and Updates` turn into `Select Tools > Extensions and Updates from the main menu.` (Install the GitHub Extension)
- [x] Fix this: `CLICK Initialize this repository with a README checkbox.` aka not all caps CLICK (Start a Project)
- [x] Fix this: `it is push your changes t` "it is time to push your changes" (Make a Commit)
- [x] Fix this: ` using git, y` capitalize Git, happens twice (What is Sync?)

### PR DESCRIPTION
**Pull request by beardofedu**
## Overview
**TL;DR**
The Windows content is up for review to ensure that the content is technically accurate and void of spelling errors (which it obviously should be 😉 ). This will also be an opportunity to identify enhancements that were suggested in the past and potentially incorporate them into the content.

Closes #573 
Closes #493 

### Review
Not ready for 👀  yet.

### CODE REVIEW

**Comment by beardofedu** [comment]
Based on the initial issue that looked to enhance the Windows content, I think my addition of how to push your changes to your repository on GitHub, as well as reviewing the content for inconsistent style usage and spelling errors makes this PR ready to be reviewed!

Looking for 👀 from @github/services-training 

### Original enhancement requests

> Example of branching, show on whiteboard, GH desktop, then within VS
> Why git 
 
This course was never designed to be a introduction to Git course, it was identified as a way for Windows users to get their system up and running with Git and GitHub. Adding this type of content just doesn't seem pertinent to the course content but willing to hear other 💭 s.

> Setup walkthrough

This exists in the course. 

> Git shell demos (show that it exists, and you don't have to login)

I guess we could add content about Git from the CLI in the Windows course, but we already have that content in the CLI course.

> Merge conflicts
> Demo beyondcompare / merge conflict resolution tools

Merge conflicts and examples of different tools are going to be shown in a different course #576

**Comment by brianamarie** [comment]
@beardofedu from the sounds of it, the changes you've made so far are ✨! I have the same comment as I had for the other course about moving the gifs to the top, but other than that, 🚢! 😍

**Comment by hollenberry** [comment]
@beardofedu I'd be happy to move the gifs to the top so you don't have to worry about it! I really appreciate you taking the reviews off of my plate, and I feel like it would make me feel better and it's the least I can do. 

I would take the initiative, but per your Slack comment I don't want to touch this if you've been working locally and haven't pushed.

**Comment by beardofedu** [comment]
I haven't done it for the Windows course yet, and can leave it for you, just check with me on Slack before you start 🔨 ing away 😁

**Comment by hollenberry** [comment]
@beardofedu All set! Planning on reviewing some time today so that we can 🚢 !

**Comment by beardofedu** [comment]
@hollenberry I just checked all the gifs and they moved without any issues. Nice work.

File: paths/windows/03f-merge-conflicts.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  1. A _merge conflict_ has occurred, click the **Conflicts: 1** option from the Team Explorer - Branches pane.
+  1. Select the README.md file and click **Merge**.
+
+      > Explore the new **Merge** window that is displayed. Two panes are displayed, **Source** and **Target**. The Source pane is based on the **Change2** branch and the Target pane is based on the **master** branch.  
+  1. Click the **Take Right** button.
+  1. Save the change by clicking the **Save** button or pressing `CTRL+S` on your keyboard.
+  1. Click **Accept Merge**. The `README.md` file now displays the changes you selected.
+
+tell-me-why: |
+  Merge conflicts happen because
```
Comment (hollenberry) [inline paths/windows/03f-merge-conflicts.md]: @beardofedu , should we populate this section, or remove it?

Comment (beardofedu) [reply paths/windows/03f-merge-conflicts.md]: This section can be removed, because it is going to be covered in #578

**Comment by beardofedu** [comment]
@hollenberry thanks for the assistance with the course. I fixed the merge conflict and tested it and it is good to go, so I am going to merge it.
